service/controller/v24/controllercontext/status.go

```diff
@@ -1,25 +1,22 @@
 package controllercontext
 
-type Status struct {
-	// Cluster carries information between cluster controller resource.
-	Cluster Cluster
-	// Drainer carries information between drainer controller resource.
-	Drainer Drainer
+type ContextStatus struct {
+	ControlPlane  ContextStatusControlPlane
+	TenantCluster ContextStatusTenantCluster
```

Comment:
I would like to have this separation. One reason is the AWS account ID. I plan to improve that handling but the controller context needs to get aligned first.

Reply:
Having the separation makes sense to me.